src/JustWatch.elm

```diff
@@ -0,0 +1,216 @@
+module JustWatch exposing (..)
+
+import Json.Decode as Decode exposing (string, list)
+import Json.Decode.Pipeline exposing (decode, hardcoded, optional, required)
+import Http exposing (encodeUri)
+
+
+type MovieOffers
+    = Loading
+    | NoResults
+    | Results (List JustWatchOffer)
+
+
+type OfferType
+    = Buy
+    | Rent
+    | Streaming
+    | Ads
+    | Free
+    | Unknown
+
+
+movieSearchUrl : String -> String
+movieSearchUrl title =
+    "https://apis.justwatch.com/content/titles/en_US/popular?body=" ++ (Http.encodeUri ("{ \"query\": \" " ++ title ++ "\"}"))
+
+
+movieDetailUrl : Int -> String
+movieDetailUrl id =
+    "https://apis.justwatch.com/content/titles/movie/" ++ (toString id) ++ "/locale/en_US"
+
+
+type alias JustWatchSearchResult =
```

Comment:
I think the typical naming convention is to try to avoid prefixing types with the module name, and instead just use the module when you import it. So instead of:

```elm
type alias JustWatchSearchResult

import JustWatch exposing (JustWatchSearchResult)

foo : JustWatchSearchResult
```
You'd do something more like:

```elm
type alias SearchResult

import JustWatch

foo : JustWatch.SearchResult
```